### PR TITLE
feat: Output Conan 1.62 package for C++ generated code

### PR DESCRIPTION
**Pull request by doosuu**
## Describe your changes

Outputs a conan package when generating c++ code.

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: src/velocitas/model_generator/__init__.py [hunk 1]
```diff
@@ -66,6 +69,9 @@ def generate_model(
         print(f"Known extended attributes: {', '.join(ext_attributes_list)}")
 
     try:
+        if os.path.exists(target_folder):
+            shutil.rmtree(target_folder)
+
```
Comment (BjoernAtBosch) [inline src/velocitas/model_generator/__init__.py:74]: Do we need this here?
Deleting target_folder is done in Python generator, also.
In C++ generator, just the folder of the root namespace is deleted, e.g.:
target_folder = "./a/b"
name = "com.acme.bla.root_ns_of_model"
-> folder of root namespace = "./a/b/include/com/acme/bla/root_ns_of_model"
This was done to enable model generation into a include folder structure with other stuff also.

File: src/velocitas/model_generator/cpp/cpp_generator.py [hunk 2]
```diff
@@ -65,6 +67,31 @@ def generate(self):
...
+        with open(path, "w", encoding="utf-8") as file:
+            file.write(
+                """from conans import ConanFile
+
+class VehicleModelConan(ConanFile):
+    name = "vehicle-model"
+    version = "generated"
+    license = "Apache 2.0"
+    url = "https://github.com/eclipse-velocitas/vehicle-model-generator"
+    description = "Vehicle Model API auto-generated from Vehicle Signal Specification"
```
Comment (BjoernAtBosch) [inline src/velocitas/model_generator/cpp/cpp_generator.py:82]: This should be reworked later to contain meaningful values

Comment (doosuu) [reply src/velocitas/model_generator/cpp/cpp_generator.py:82]: We will do that later, as discussed.

**Review by BjoernAtBosch — APPROVED** [review]
LGTM

**Pull request merged**